editor.html

```diff
@@ -74,40 +74,48 @@
             UPY_VERSION = "1.0.1";
         </script>
         <script id="load-template" type="x-tmpl-mustache">
-            <h2><i class="fa fa-upload"></i> <strong>{{ load-title }}</strong></h2>
+            <h2 style="margin-top: 0;"><i class="fa fa-download"></i> <strong>{{ load-title }}</strong></h2>
             <div class="load-drag-target" id="load-drag-target">
                 <input type="file" style="display: none" name="load-form-file-upload" id="file-upload-input">
                 <p>{{ instructions }} <a href="#" id="file-upload-link" class="load-drag-target load-toggle action">{{ toggle-file }}</a></p>
             </div>
-
-            <h2><i class="fa fa-folder"></i> <strong>{{ fs-title }}</strong></h2>
-            <table class='fs-space-table'>
-                <tbody>
-                  <tr>
-                    <td id="fs-space-modules" class="fs-space-table-first">modules</td>
-                    <td id="fs-space-main" title="main.py">main.py</td>
-                    <td id="fs-space-other" title="other">other</td>
-                    <td id="fs-space-free" title="free">free</td>
-                  </tr>
-                </tbody>
-            </table>
-            <div class="fs-file-list" id="fs-file-list">
-                <table>
-                    <thead>
-                        <tr>
-                            <td>Filename</td>
-                            <td>Type</td>
-                            <td>Size</td>
-                            <td></td>
-                        </tr>
-                    </thead>
+            <hr style="border-top: dashed 3px; color: lightgray;" />
+            <div class="expandable">
+              <div>
+                  <h3 style="display: inline-block; margin: 12px 0 0 2px;">{{ program-title }}</h3>
+                  <button style="margin-top: 2px; float: right; width: 4em;" type="button" class="action save-button" id="save-hex" title="{{ save-hex }}"><i class="fa fa-download"></i></button>
+                  <button style="margin-top: 2px; float: right;" type="button" class="action save-button" id="show-files" title="{{ show-files }}">{{ show-files }} <i class="fa fa-caret-down"></i></button>
+              </div>
+              <div class="content">
+                <div class="fs-file-list" id="fs-file-list">
+                    <table>
```

Comment:
This table hard round corners in python.microbit.org/v/beta were those removed on purpose?
I would keep round corners on this table with a lower radius than the wrapper box.

Reply:
Yes I removed them as it was quite jarring having two levels of rounded corners

Reply:
With a bit more padding and a lower radius to me this looks a bit better, but it's all subjective:
![image](https://user-images.githubusercontent.com/29712657/58641028-4c946180-82f2-11e9-8e9e-43f6ab042a38.png)